app/models/one_login/core_identity_validator.rb

```diff
@@ -21,6 +21,10 @@ def date_of_birth
     Date.parse(decoded_jwt[0]["vc"]["credentialSubject"]["birthDate"][0]["value"])
   end
 
+  def full_name
+    name_parts.map { |hash| hash["value"] }.join(" ")
```

Comment:
Should we ensure that the `FamilyName` part(s) are always last? I suspect there's no guarantee it will be ordered this way already.